pyproject.toml

```diff
@@ -91,8 +91,3 @@ exclude = '''
   | tests/data
 )/
 '''
-
-[tool.codespell]
```

Comment:
any reason codespell is being removed?

Reply:
I see, there is another `[tool.codspell]` defined in this pyprojec.toml file.

Reply:
yes it's a duplicate!